tensorflow_addons/metrics/cohens_kappa.py

```diff
@@ -38,28 +38,30 @@ class CohenKappa(Metric):
 
     Usage:
 
-    ```python
-    actuals = np.array([4, 4, 3, 4, 2, 4, 1, 1], dtype=np.int32)
-    preds = np.array([4, 4, 3, 4, 4, 2, 1, 1], dtype=np.int32)
-    weights = np.array([1, 1, 2, 5, 10, 2, 3, 3], dtype=np.int32)
 
-    m = tfa.metrics.CohenKappa(num_classes=5, sparse_labels=True)
-    m.update_state(actuals, preds)
-    print('Final result: ', m.result().numpy()) # Result: 0.61904764
+    >>> actuals = np.array([4, 4, 3, 4, 2, 4, 1, 1], dtype=np.int32)
+    >>> preds = np.array([4, 4, 3, 4, 4, 2, 1, 1], dtype=np.int32)
+    >>> weights = np.array([1, 1, 2, 5, 10, 2, 3, 3], dtype=np.int32)
 
-    # To use this with weights, sample_weight argument can be used.
-    m = tfa.metrics.CohenKappa(num_classes=5, sparse_labels=True)
-    m.update_state(actuals, preds, sample_weight=weights)
-    print('Final result: ', m.result().numpy()) # Result: 0.37209308
-    ```
+    >>> m = tfa.metrics.CohenKappa(num_classes=5, sparse_labels=True)
+    >>> # m.update_state(actuals, preds)
+    >>> # print('Final result: ', m.result().numpy())
+    >>> # Final result: 0.61904764
+
+    >>> # To use this with weights, sample_weight argument can be used.
+    >>> m = tfa.metrics.CohenKappa(num_classes=5, sparse_labels=True)
+    >>> # m.update_state(actuals, preds, sample_weight=weights)
+    >>> # print('Final result: ', m.result().numpy())
+    >>> # Final result: 0.37209308
```

Comment:
ditto

Reply:
it's not complete ditto the second example uses weights.